deploy/deploy-dev.sh

```
@@ -0,0 +1,49 @@
+PROFILE="dev"
+APP_DIR="/home/ubuntu/forgather-backend"
+LOG_DIR="$APP_DIR/logs"
+JAR_PATH="$APP_DIR/$JAR_NAME"
+NOHUP_LOG="$LOG_DIR/nohup.log"
+
+echo "기존 $PORT 포트 프로세스 종료 시도..."
+PID=$(sudo lsof -t -i:$PORT)
+
+if [ -n "$PID" ]; then
+  echo "PID $PID 종료 중..."
+  kill -9 $PID
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)